variables.tf

```diff
@@ -71,6 +71,6 @@ variable "common_tags" {
 
 variable "allowed_jurisdictions" {
   type        = string
-  default = "'ia', 'IA', 'sscs', 'SSCS', 'wa', 'WA', 'civil', 'CIVIL', 'publiclaw', 'PUBLICLAW', 'privatelaw', 'PRIVATELAW', 'employment','EMPLOYMENT'"
+  default = "'ia', 'IA', 'civil', 'CIVIL'"
```

Comment:
Looks good.  I think these are the only 2 teams with an AtO currently.

Reply:
yes, that is correct.